Tabs.php

```diff
@@ -290,4 +290,19 @@ public function renderPanes($panes)
     {
         return $this->renderTabContent ? "\n" . Html::tag('div', implode("\n", $panes), ['class' => 'tab-content']) : '';
     }
+    /**
+     * Sets the first visible tab as active
+     * @since 2.0.7
+     */
+    protected function activateTab()
```

Comment:
Thinking about it more, I'd rename it to `activateFirstVisibleTab()`. More descriptive...